### DISCUSSION TITLE
otlphttp exporter

### DISCUSSION THREAD

**Discussion by klempinen** [open-telemetry/opentelemetry-go#3248] [Q&A] [answered]
Hi

I am trying to use otlphttp exporter and once I send the data to APM backend, I seem to get this error

`2022/09/30 14:13:03 proto: cannot parse invalid wire-format data`

Any idea what could be causing this as same backend works fine with other platforms? Also Is there way to print server response?

Thanks, Kim 

ps. I am new to Go

**Comment by MrAlias** [comment]
Can you please add a minimal code example to reproduce the error you are seeing? Your question is very ambiguous without it.

**Reply by klempinen** [reply]
I found the reason myself, this one can be considered resolved.

**Reply by dmathieu** [reply]
Could you describe what this was, for anybody who would visit this discussion in the future?

**Reply by gmreads** [reply]
I got this error too, because my configured receiver didn't understand protobuff, was replying with JSON and 400.
OTLP supported backends like Jaeger work just fine

**Comment by pellared** [comment, marked as answer]
By @gmreads 

> I got this error too, because my configured receiver didn't understand protobuff, was replying with JSON and 400.
OTLP supported backends like Jaeger work just fine
